docs/faq.md

```diff
@@ -2,119 +2,160 @@
+- Excerpts for subheads.
+- Widgets for content on the side of a page.
+
+As we thought about these uses and how to make them obvious and consistent, we began to embrace the concept of “blocks.” All of the above items could be blocks: easy to search and understand, and easy to dynamically shift around the page. The block concept is very powerful, and if designed thoughtfully, can offer an outstanding editing and publishing experience.
+
+## What is the writing experience like?
+
+Our goal with Gutenberg is not just to create a seamless post- and page-building experience. We also want to ensure that it provides a seamless writing experience. Though individual paragraphs of text will become their own “blocks,” the creation and editing of these blocks are being designed in a way that could be just as simple—if not more so—than the current WordPress editor experience. Here is a brief animation illustrating the Gutenberg writing experience:
+
+https://make.wordpress.org/core/files/2017/08/typing.gif
 
 ## Is Gutenberg built on top of TinyMCE?
-No.
 
-TinyMCE is one of the best tools for enabling rich text on the web. In Gutenberg, TinyMCE does exactly that. Nearly every textfield you'll find is augmented with TinyMCE for rich text. Whether it be text, lists, or even just a single caption, TinyMCE can be invoked on blocks for rich text enhancements.
+No. TinyMCE is one of the best tools for enabling rich text on the web. In Gutenberg, TinyMCE does exactly that. Nearly every text field you'll find is augmented with TinyMCE for rich text. Whether it be text, lists, or even just a single caption, TinyMCE can be invoked on blocks for rich text enhancements.
 
 ## What browsers will Gutenberg support?
 
 Gutenberg will work in modern browsers, and Internet Explorer 11.
 
 ## How do I make my own block?
-The API for creating blocks is a crucial aspect of the project. We are working on improved documentation and tutorials. Current documentation lives here: https://github.com/WordPress/gutenberg/tree/master/blocks
 
-## Does Gutenberg involve editing posts/pages in the frontend?
-No, Gutenberg is a replacement for the post and page editing screens.
+The API for creating blocks is a crucial aspect of the project. We are working on improved documentation and tutorials. Current documentation lives here
+
+## Does Gutenberg involve editing posts/pages in the front-end?
+
+No, we are designing Gutenberg primarily as a replacement for the post and page editing screens. That said, front-end editing is often confused with an editor that looks exactly like the front end. And that is something that Gutenberg will allow as themes customize individual blocks and provide those styles to the editor. Since content is designed to be distributed across so many different experiences—from desktop and mobile to full-text feeds and syndicated article platforms—we believe it's not ideal to create or design posts from just one front-end experience.
```

Comment:
Do we care to make any mention of editor as a set of reusable pieces that someone could potentially use in a front-end context the way they might be envisioning?

Reply:
Good idea.